### PR TITLE
Use new npdm-json format

### PR DESCRIPTION
**Pull request by roblabla**
# Description

The current NPDM format is going to change in the near future. In order to make it easier for the homebrew ecosystem to move to the new format, PRs have been issued automatically to upgrade your JSON to the new format. See https://github.com/switchbrew/switch-tools/pull/20 for more information.

# What you need to do

Right now, nothing. The PR to npdmtool is written in such a way that, when it's merged, both the old and new JSON formats will be accepted. Once it is merged and a new version of switch-tools has been published on the devkitPro repositories, you should merge this PR. I will leave a comment here when this is the case.

### CODE REVIEW

File: hbl.json [hunk 1]
```diff
@@ -1,142 +1,159 @@
...
+    "pool_partition": 0,
+    "is_64_bit": true,
+    "address_space_type": 1,
+    "is_retail": true,
+    "filesystem_access": {
+        "permissions": "0xFFFFFFFFFFFFFFFF"
     },
-    "service_access" : {
-        "*" : false,
-        "*" : true
```
Comment (yellows8) [inline hbl.json]: nx-hbloader needs both...

Comment (roblabla) [reply hbl.json]: Ugh. I totally missed that. This is another case of duplicated keys...

Comment (roblabla) [reply hbl.json]: Alright, we're going to split this into `"service_access": []` and `"service_host": []`

Comment (yellows8) [reply hbl.json]: sure

**Comment by roblabla** [comment]
Should be fixed now.

**Pull request merged**